### ISSUE TITLE
yarn install errors

### ISSUE THREAD

**Issue by SergeyDjam** [mattermost/desktop#2367]
Mattermost v5.2.0 tar.gz github

yarn install
```
[5/5] Building fresh packages...
success Saved lockfile.
$ node scripts/patch_macos_notification_state.js && electron-builder install-app-deps
node:internal/modules/cjs/loader:959
  throw err;
  ^

Error: Cannot find module 'node-jq'
Require stack:
- /home/builder/rpmbuild/BUILD/desktop-5.2.0/scripts/patch_macos_notification_state.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/builder/rpmbuild/BUILD/desktop-5.2.0/scripts/patch_macos_notification_state.js:6:12)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/builder/rpmbuild/BUILD/desktop-5.2.0/scripts/patch_macos_notification_state.js'
  ]
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

yarn --version
1.22.19

node 16.17.0
ROSA Linux Fresh r11.1

I confirm (by marking "x" in the [ ] below: [x]):

- [ ] This is not a troubleshooting question. [Troubleshooting questions go here: https://docs.mattermost.com/install/troubleshooting.html](https://docs.mattermost.com/install/troubleshooting.html).
- [ ] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [ ] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

---

**Summary**

**Environment**
- Operating System:
- Mattermost Desktop App version: X.X.X 
- Mattermost Server version: X.X.X 

**Steps to reproduce**

**Expected behavior**

**Observed behavior**

**Possible fixes**

**Comment by devinbinnie** [comment]
We don't currently support `yarn`, if you use node v16+ and npm v8+ and use `npm install` that should work correctly.

**Comment by SergeyDjam** [comment]
Hello. 

node --version
v16.17.0

npm install

`npm WARN deprecated xmldom@0.1.31: Deprecated due to CVE-2021-21366 resolved in 0.5.0

> mattermost-desktop@5.2.2-rc.2 postinstall
> node scripts/patch_macos_notification_state.js && electron-builder install-app-deps

node:internal/modules/cjs/loader:959
  throw err;
  ^

Error: Cannot find module 'node-jq'
Require stack:
- /home/builder/rpmbuild/BUILD/desktop-5.2.2-rc.2/scripts/patch_macos_notification_state.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/builder/rpmbuild/BUILD/desktop-5.2.2-rc.2/scripts/patch_macos_notification_state.js:6:12)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/builder/rpmbuild/BUILD/desktop-5.2.2-rc.2/scripts/patch_macos_notification_state.js'
  ]
}
npm notice 
npm notice New major version of npm available! 8.15.0 -> 9.1.3
npm notice Changelog: https://github.com/npm/cli/releases/tag/v9.1.3
npm notice Run npm install -g npm@9.1.3 to update!
npm notice 
npm ERR! code 1
npm ERR! path /home/builder/rpmbuild/BUILD/desktop-5.2.2-rc.2
npm ERR! command failed
npm ERR! command sh /tmp/postinstall-cd21d2e1.sh

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/builder/.npm/_logs/2022-12-01T02_32_53_474Z-debug-0.log
error: Bad exit status from /var/tmp/rpm-tmp.85309 (%build)
`

npm install -u npm@9.1.3
npm install

`To address issues that do not require attention, run:
  npm audit fix

To address all issues, run:
  npm audit fix --force

Run `npm audit` for details.
+ npm install

> mattermost-desktop@5.2.2-rc.2 postinstall
> node scripts/patch_macos_notification_state.js && electron-builder install-app-deps

node:internal/modules/cjs/loader:959
  throw err;
  ^

Error: Cannot find module 'node-jq'
Require stack:
- /home/builder/rpmbuild/BUILD/desktop-5.2.2-rc.2/scripts/patch_macos_notification_state.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/builder/rpmbuild/BUILD/desktop-5.2.2-rc.2/scripts/patch_macos_notification_state.js:6:12)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/builder/rpmbuild/BUILD/desktop-5.2.2-rc.2/scripts/patch_macos_notification_state.js'
  ]
}
npm ERR! code 1
npm ERR! path /home/builder/rpmbuild/BUILD/desktop-5.2.2-rc.2
npm ERR! command failed
npm ERR! command sh /tmp/postinstall-a493f141.sh

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/builder/.npm/_logs/2022-12-01T02_41_40_576Z-debug-0.log
error: Bad exit status from /var/tmp/rpm-tmp.45340 (%build)
`

mattermost-desktop 5.2.2-rc.2

**Comment by SergeyDjam** [comment]
> We don't currently support `yarn`, if you use node v16+ and npm v8+ and use `npm install` that should work correctly.

That doesn't solve the problem.
